packages/generator/src/v2.0/generateClientSdkVersionTwoPointZero.test.ts

```diff
@@ -1683,14 +1690,13 @@ describe("generator", () => {
           helper.minimalFiles,
           BASE_PATH,
           "module",
-          "foo.bar",
           new Map(),
         ),
       ).resolves.toMatchInlineSnapshot(`undefined`);
 
-      expect(helper.getFiles()["/foo/index.ts"]).not.toContain(
-        "$ontologyRid",
-      );
+      // expect(helper.getFiles()["/foo/index.ts"]).not.toContain(
```

Comment:
Want to just delete this for now?

Reply:
I really wish this constraint could come back